### PR TITLE
adds cloudsql example

### PR DESCRIPTION
**Pull request by bshaffer**

### CODE REVIEW

File: appengine/standard/cloudsql/app.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+$app->post('/', function (Request $request) use ($app) {
+    /** @var PDO $db */
+    $db = $app['database'];
+
+    $name = $request->request->get('name');
+    $content = $request->request->get('content');
+
+    if ($name && $content) {
+        $stmt = $db->prepare('INSERT INTO entries (guestName, content) VALUES (:name, :content)');
```
Comment (JustinBeckwith) [inline appengine/standard/cloudsql/app.php]: Not knowing PHP very well - are we sql injection attack safe here?

**Comment by JustinBeckwith** [comment]
LGTM

File: appengine/standard/cloudsql/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+composer install
+```
+
+## Setup
+
+Before you can run or deploy the sample, you will need to do the following:
+
+1. Create a [First Generation Cloud SQL](https://cloud.google.com/sql/docs/create-instance) instance. You can do this from the [Cloud Console](https://console.developers.google.com) or via the [Cloud SDK](https://cloud.google.com/sdk). To create it via the SDK use the following command:
+
+        $ gcloud sql instances create YOUR_INSTANCE_NAME
```
Comment (tmatsuo) [inline appengine/standard/cloudsql/README.md]: Does it create a first generation instance?

File: appengine/standard/cloudsql/app.php (hunk 1 above)
Comment (tmatsuo) [inline appengine/standard/cloudsql/app.php]: it's safe :)

**Comment by tmatsuo** [comment]
LGTM

File: appengine/standard/cloudsql/app.php (hunk 1 above)
Comment (bshaffer) [inline appengine/standard/cloudsql/app.php]: @JustinBeckwith Good question! PDO [prepared statements](http://php.net/manual/en/pdo.prepared-statements.php) are sufficient to protect against SQL injection: 

    If an application exclusively uses prepared statements, the developer can be sure that no SQL injection will occur

File: appengine/standard/cloudsql/README.md (hunk 2 above)
Comment (bshaffer) [inline appengine/standard/cloudsql/README.md]: Yes

**Pull request merged**